backend/typescript/graphql/resolvers/warningResolvers.ts

```diff
@@ -0,0 +1,30 @@
+import WarningService from "../../services/implementations/warningService";
+import type {
+  WarningDTO,
+  CreateWarningDTO,
+  IWarningService,
+} from "../../services/interfaces/warningService";
+
+const warningService: IWarningService = new WarningService();
+// const authService: IAuthService = new AuthService(userService, emailService);
+
+const warningResolvers = {
+  Mutation: {
+    addWarning: async (
+      _parent: undefined,
+      { warning }: { warning: CreateWarningDTO },
+    ): Promise<WarningDTO> => {
+      const newWarning = await warningService.addWarning(warning);
+      return newWarning;
+    },
+    deleteWarning: async (
+      _parent: undefined,
+      { id }: { id: number },
+    ): Promise<WarningDTO> => {
+      const deletedWarning = await warningService.deleteWarning(Number(id));
```

Comment:
`id` does not need to be casted into a number, since it's already being defined as a number on line 22

Reply:
in warningTypes, we use ID, not Int, so we need to make it a Number. Either that or delete the quotation marks in postman every time